src/trace-plugin-interface.js

```diff
@@ -102,7 +102,6 @@ RootSpan.prototype.getTraceContext = function() {
 /**
  * PluginAPI constructor. Don't call directly - a plugin object will be passed to
  * plugin themselves
- * TODO(kjin): Should be called something else
```

Comment:
This renaming will occur in https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/pull/411. No longer blocked.